package-lock.json

```diff
@@ -1,44 +1,92 @@
 {
   "name": "wait-on",
   "version": "7.2.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
Please check if updating the `lockfileVersion` to `3` would make it a breaking change.
The README claims compatibility for Node.js 12+
Node.js `12` however uses npm `6` which in turn uses `lockfileVersion` `1`
https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json#lockfileversion

Reply:
My bad, my default node version is 18, didn't see the change in the `lockfileVersion`.